products/memgraph/v0.51.0/reference_guide/query-modules.md

```diff
@@ -99,18 +99,42 @@ offering.
 * `louvain` [Enterprise]: Louvain algorithm for community detection.
 * `connectivity` [Enterprise]: Algorithms that analyse graph connectivity.
 
-At the moment, these modules offer the following query procedures:
+In addition to low-level modules listed above, Memgraph Community offering
+provides the following Python modules based on
+[NetworkX](https://networkx.github.io/) algorithms.
+
+* `graph_analyzer`: Module that offers more insights about the stored graph. To
+  get a detailed list of provided functionalities within this module run
+`CALL graph_analyzer.help() YIELD *;`.
+* `pagerank`: Page Rank algorithm for centrality calculations.
+* `wcc`: Module that offers analysis of weakly connected components.
+
+The utility module offers the following functionality:
 
 * `mg.procedures() :: (name :: STRING, signature :: STRING)`: lists loaded
-  procedures and their signatures
-* `mg.reload(module_name :: STRING) :: ()`: reloads the given module
-* `mg.reload_all() :: ()`: reloads all loaded modules
+  procedures and their signatures.
+* `mg.reload(module_name :: STRING) :: ()`: reloads the given module.
+* `mg.reload_all() :: ()`: reloads all loaded modules.
```

Comment:
Because you changed the descriptions to end with a `.`, change them to have a capital first letter (as you have in the descriptions above).